qcengine/programs/nwchem/runner.py

```diff
@@ -154,6 +154,9 @@ def build_input(
         molcmd, moldata = input_model.molecule.to_string(dtype="nwchem", units="Bohr", return_data=True)
         opts.update(moldata["keywords"])
 
+        if opts.pop("noautoz", False):
```

Comment:
what do you think of `geometry__noautoz` for the option name? it'd help with consistency elsewhere.

Reply:
I like that better. Makes sense along with the other NWChem settings. I'll make the change :)